/* eslint quotes: off */

import { Gene, GCTGene } from '../models';

export const mockGene1: Gene = {
  _id: '628ea1be0e8d04279fdbaa26',
  ensembl_gene_id: 'ENSG00000147065',
  name: 'moesin',
  summary:
    'Moesin (for membrane-organizing extension spike protein) is a member of the ERM family which includes ezrin and radixin. ERM proteins appear to function as cross-linkers between plasma membranes and actin-based cytoskeletons. Moesin is localized to filopodia and other membranous protrusions that are important for cell-cell recognition and signaling and for cell movement. [provided by RefSeq, Jul 2008].',
  hgnc_symbol: 'MSN',
  alias: ['HEL70', 'IMD50'],
  isIGAP: false,
  haseqtl: false,
  isAnyRNAChangedInADBrain: true,
  rna_brain_change_studied: true,
  isAnyProteinChangedInADBrain: true,
  protein_brain_change_studied: true,
  nominatedtarget: [
    {
      source: 'Community',
      team: 'Chang Lab',
      rank: 'PS4-1',
      ensembl_gene_id: 'ENSG00000147065',
      hgnc_symbol: 'MSN',
      target_choice_justification:
        'MSN was identified as an important key driver by predictive network analysis in dorsolateral prefrontal cortex in ROSMAP cohort, and frontal and temporal cortex in KRONOSII cohort, and Brodmann area 10 in MSBB cohort.',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression',
      data_used_to_support_target_selection:
        'Predictive network analysis derived from RNAseq, Proteomics and GWAS genotypes',
      data_synapseid: 'syn18358612',
      study: 'ROSMAP, Kronos, MSBB',
      input_data: 'Genetics, RNA, Protein',
      validation_study_details: '',
      initial_nomination: 2019,
    },
    {
      source: 'AMP-AD',
      team: 'Emory',
      rank: '7',
      ensembl_gene_id: 'ENSG00000147065',
      hgnc_symbol: 'MSN',
      target_choice_justification:
        'MSN was identified as a potential driver protein based on protein coexpression analysis. The group of proteins coexpressed with MSN is conserved across the 3 datasets considered, is enriched for inflammatory processes, and for protein products of genes near loci previously associated with AD risk. MSN has increased abundance in AD across all 3 cohorts examined, and progressively increases in asymptomatic (prodromal) AD to symptomatic AD, and also correlates with both hallmark AD pathology scores (CERAD for amyloid burden; and Braak for Tau extent of spread). MSN is highly expressed as a marker of disease-associated microglia and/or endothelial cell types.',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression. Phosphorylation downstream of Rho/Rock influences actin, focal adhesion binding; may have redundancy with EZR and RDX, complicating targeting. MSN-directed therapeutics that improve microglial motility and/or phagocytosis competence would reduce abeta/amyloid plaque burden.',
      data_used_to_support_target_selection:
        'Discovery quantitative proteomics of FrCx \r\n WPCNA of multiple and consensus cohorts\r\n ANOVA',
      data_synapseid: 'syn17008058',
      study: 'ACT, BLSA, Banner',
      input_data: 'Protein',
      validation_study_details: 'validation studies ongoing',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'MSSM',
      rank: '35',
      ensembl_gene_id: 'ENSG00000147065',
      hgnc_symbol: 'MSN',
      target_choice_justification:
        'MSN up-regulated in the parahippocampal area of CERAD definite AD brains. It was identified in the Bayesian network analysis as an important key driver of up-regulation of genes associated with AD severity.',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression. MSN was significantly up-regulated in the parahippocampal area of AD (p=6.943054e-07, FC=1.351137) / Demented (p=4.371646e-05, FC=1.372884) / severly Plaqued (p=5.965625e-06, FC=1.422039) brains compared to the normal brains.',
      data_used_to_support_target_selection:
        'Bayesian network analysis derived from RNAseq and WGS genotypes.',
      data_synapseid: 'syn17008057',
      study: 'MSBB',
      input_data: 'Genetics, RNA',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'MSSM',
      rank: '1',
      ensembl_gene_id: 'ENSG00000147065',
      hgnc_symbol: 'MSN',
      target_choice_justification:
        'The protein expression of MSN was up-regulated in the brain parahippocampal area in AD vs. NL across various AD clinic traits (Braak score, CDR, CERAD, Plaque-mean). It was identified as an important key driver protein in the Bayesian network analysis.',
      predicted_therapeutic_direction:
        'Agonism predicted to reduce disease progression. MSN silencing by siRNA decreased Aï¿½ phagocytosis and increased lipopolysaccharide-induced production of the pro-inflammatory cytokine, tumor necrosis factor (TNF)',
      data_used_to_support_target_selection:
        'Differential protein expression analysis, and Bayesian network analysis derived from proteomics profiling and WGS genotypes.',
      data_synapseid: 'syn17008057',
      study: 'MSBB',
      input_data: 'Genetics, Protein',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2020,
    },
  ],
  medianexpression: [
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 5.04336509575029,
      quartile1logcpm: 6.41020176277925,
      medianlogcpm: 6.73621709567464,
      meanlogcpm: 6.7475379442254,
      quartile3logcpm: 7.05899186515986,
      maximumlogcpm: 8.83145510754094,
      tissue: 'CBE',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 5.93408536362101,
      quartile1logcpm: 6.96615507794204,
      medianlogcpm: 7.38240334905122,
      meanlogcpm: 7.4311228447336,
      quartile3logcpm: 7.92560049264113,
      maximumlogcpm: 9.18471815917279,
      tissue: 'TCX',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 6.23306725775568,
      quartile1logcpm: 6.90433999850259,
      medianlogcpm: 7.14860056475605,
      meanlogcpm: 7.21193702176113,
      quartile3logcpm: 7.4621484031443,
      maximumlogcpm: 9.34971847444808,
      tissue: 'ACC',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 4.28165355601,
      quartile1logcpm: 5.8367467879018,
      medianlogcpm: 6.28136740929578,
      meanlogcpm: 6.32530808628807,
      quartile3logcpm: 6.7394655622644,
      maximumlogcpm: 8.74464268762163,
      tissue: 'DLPFC',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 4.85658305980558,
      quartile1logcpm: 6.14158577569289,
      medianlogcpm: 6.56845455430893,
      meanlogcpm: 6.6639844809369,
      quartile3logcpm: 7.05759596620699,
      maximumlogcpm: 8.87327023680568,
      tissue: 'PCC',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 4.26404404037343,
      quartile1logcpm: 5.01578322637647,
      medianlogcpm: 5.48704289453951,
      meanlogcpm: 5.66980171681199,
      quartile3logcpm: 6.31002425154561,
      maximumlogcpm: 8.63399008131042,
      tissue: 'FP',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 5.16157306923804,
      quartile1logcpm: 6.11769646241097,
      medianlogcpm: 6.47524443721823,
      meanlogcpm: 6.54955150483383,
      quartile3logcpm: 6.95630227736829,
      maximumlogcpm: 9.09785223163572,
      tissue: 'IFG',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 4.07237364602701,
      quartile1logcpm: 5.58783452541112,
      medianlogcpm: 6.26207347848023,
      meanlogcpm: 6.2369971707934,
      quartile3logcpm: 6.95874299676971,
      maximumlogcpm: 8.00698573745332,
      tissue: 'PHG',
    },
    {
      ensembl_gene_id: 'ENSG00000147065',
      minimumlogcpm: 4.30589104835697,
      quartile1logcpm: 5.40773593449646,
      medianlogcpm: 6.3210556430281,
      meanlogcpm: 6.22552260388434,
      quartile3logcpm: 6.86982064764693,
      maximumlogcpm: 9.0312360634679,
      tissue: 'STG',
    },
  ],
  druggability: [
    {
      geneid: 'ENSG00000147065',
      sm_druggability_bucket: 3,
      safety_bucket: 4,
      abability_bucket: 3,
      pharos_class: 'Tbio',
      classification:
        'Targetable by structure: Structurally druggable protein, based on the presence of a druggable pocket in the protein (DrugEBIlity/CanSAR).',
      safety_bucket_definition:
        'More than two of: high off target gene expression, cancer driver, essential gene, associated deleterious genetic disorder, HPO phenotype associated gene, or black box warning on clinically used drug.',
      abability_bucket_definition:
        'Cell membrane-bound proteins. Highly accessible to antibody-based therapies, but potentially less so than secreted proteins or ECM components.',
    },
  ],
  nominations: 4,
};

export const mockGene2: Gene = {
  _id: '628ea1c10e8d04279fdbbfbc',
  ensembl_gene_id: 'ENSG00000178209',
  name: 'plectin',
  summary:
    "Plectin is a prominent member of an important family of structurally and in part functionally related proteins, termed plakins or cytolinkers, that are capable of interlinking different elements of the cytoskeleton. Plakins, with their multi-domain structure and enormous size, not only play crucial roles in maintaining cell and tissue integrity and orchestrating dynamic changes in cytoarchitecture and cell shape, but also serve as scaffolding platforms for the assembly, positioning, and regulation of signaling complexes (reviewed in PMID: 9701547, 11854008, and 17499243). Plectin is expressed as several protein isoforms in a wide range of cell types and tissues from a single gene located on chromosome 8 in humans (PMID: 8633055, 8698233). Until 2010, this locus was named plectin 1 (symbol PLEC1 in human; Plec1 in mouse and rat) and the gene product had been referred to as 'hemidesmosomal protein 1' or 'plectin 1, intermediate filament binding 500kDa'. These names were superseded by plectin. The plectin gene locus in mouse on chromosome 15 has been analyzed in detail (PMID: 10556294, 14559777), revealing a genomic exon-intron organization with well over 40 exons spanning over 62 kb and an unusual 5' transcript complexity of plectin isoforms. Eleven exons (1-1j) have been identified that alternatively splice directly into a common exon 2 which is the first exon to encode plectin's highly conserved actin binding domain (ABD). Three additional exons (-1, 0a, and 0) splice into an alternative first coding exon (1c), and two additional exons (2alpha and 3alpha) are optionally spliced within the exons encoding the acting binding domain (exons 2-8). Analysis of the human locus has identified eight of the eleven alternative 5' exons found in mouse and rat (PMID: 14672974); exons 1i, 1j and 1h have not been confirmed in human. Furthermore, isoforms lacking the central rod domain encoded by exon 31 have been detected in mouse (PMID:10556294), rat (PMID: 9177781), and human (PMID: 11441066, 10780662, 20052759). The short alternative amino-terminal sequences encoded by the different first exons direct the targeting of the various isoforms to distinct subcellular locations (PMID: 14559777). As the expression of specific plectin isoforms was found to be dependent on cell type (tissue) and stage of development (PMID: 10556294, 12542521, 17389230) it appears that each cell type (tissue) contains a unique set (proportion and composition) of plectin isoforms, as if custom-made for specific requirements of the particular cells. Concordantly, individual isoforms were found to carry out distinct and specific functions (PMID: 14559777, 12542521, 18541706). In 1996, a number of groups reported that patients suffering from epidermolysis bullosa simplex with muscular dystrophy (EBS-MD) lacked plectin expression in skin and muscle tissues due to defects in the plectin gene (PMID: 8698233, 8941634, 8636409, 8894687, 8696340). Two other subtypes of plectin-related EBS have been described: EBS-pyloric atresia (PA) and EBS-Ogna. For reviews of plectin-related diseases see PMID: 15810881, 19945614. Mutations in the plectin gene related to human diseases should be named based on the position in NM_000445 (variant 1, isoform 1c), unless the mutation is located within one of the other alternative first exons, in which case the position in the respective Reference Sequence should be used. [provided by RefSeq, Aug 2011].",
  hgnc_symbol: 'PLEC',
  alias: [
    'EBS1',
    'EBS5A',
    'EBS5B',
    'EBS5C',
    'EBS5D',
    'EBSMD',
    'EBSND',
    'EBSO',
    'EBSOG',
    'EBSPA',
    'HD1',
    'LGMD2Q',
    'LGMDR17',
    'PCN',
    'PLEC1',
    'PLEC1b',
    'PLTN',
  ],
  isIGAP: false,
  haseqtl: true,
  isAnyRNAChangedInADBrain: true,
  rna_brain_change_studied: true,
  isAnyProteinChangedInADBrain: true,
  protein_brain_change_studied: true,
  nominatedtarget: [
    {
      source: 'Community',
      team: 'Chang Lab',
      rank: 'PS1-19',
      ensembl_gene_id: 'ENSG00000178209',
      hgnc_symbol: 'PLEC',
      target_choice_justification:
        'PLEC was identified as an important key driver by predictive network analysis in Brodmann area 10 in MSBB cohort.',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression',
      data_used_to_support_target_selection:
        'Predictive network analysis derived from Proteomics and GWAS genotypes',
      data_synapseid: 'syn18358612',
      study: 'MSBB',
      input_data: 'Genetics, Protein',
      validation_study_details: '',
      initial_nomination: 2019,
    },
    {
      source: 'AMP-AD',
      team: 'Columbia-Rush',
      rank: '17-100',
      ensembl_gene_id: 'ENSG00000178209',
      hgnc_symbol: 'PLEC',
      target_choice_justification:
        'Highly connected (hub) protein in a module enriched with proteins related to extracellular processes, which are on average related to cognition, amyloid plaque and tau tangle counts. This individual protein is associated with cognition (p<.001).',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression.',
      data_used_to_support_target_selection: 'TMT proteomics (partial dataset)',
      data_synapseid: 'syn17008059',
      study: 'ROSMAP',
      input_data: 'Protein, Clinical',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'Emory',
      rank: '3',
      ensembl_gene_id: 'ENSG00000178209',
      hgnc_symbol: 'PLEC',
      target_choice_justification:
        "PLEC was identified as a potential driver protein based on protein coexpression analysis. Driver candidate status survives regression or removal of cell type bias. The group of proteins coexpressed with PLEC is conserved across the 3 datasets considered, is enriched for inflammatory processes, and for protein products of genes near loci previously associated with AD risk. PLEC has increased abundance in AD across all 3 cohorts examined, and progressively increases in asymptomatic (prodromal) AD to symptomatic AD, and also correlates with both hallmark AD pathology scores (CERAD for amyloid burden; and Braak for Tau extent of spread). This protein's mRNA is considered to be low in brain compared to other tissues based on public data. Notably, one PLEC protein isoform is coexpressed with a different group of axon-marker-enriched proteins, with AD-dependence implicating unique splicing changes potentially in neurons.",
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression. That is, reduce this module hub\'s expression to restore normal motility of various cell types (appears to be cell-type dependent); PLEC may be a bystander hub since our hubs have no inferred (Bayesian) directionality and are therefore not quite correctly called "key drivers", but if a driver, its reduction would normalize heterotypic cell-cell interactions in the CNS which when dysregulated may induce disease-associated phenotypes, e.g. in microglia (DAM, disease-associated microglia) and/or astrocytes.',
      data_used_to_support_target_selection:
        'Discovery quantitative proteomics of FrCx \r\n WPCNA of multiple and consensus cohorts\r\n ANOVA',
      data_synapseid: 'syn17008058',
      study: 'ACT, BLSA, Banner',
      input_data: 'Protein',
      validation_study_details: 'validation studies ongoing',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'MSSM',
      rank: '6',
      ensembl_gene_id: 'ENSG00000178209',
      hgnc_symbol: 'PLEC',
      target_choice_justification:
        'The protein expression of PLEC was up-regulated in the brain parahippocampal area in AD vs. NL across various AD clinic traits (Braak score, CDR, CERAD, Plaque-mean). It was identified as an important key driver protein in the Bayesian network analysis.',
      predicted_therapeutic_direction:
        'Antagonism predicted to reduce disease progression. Up-regulation of PLEC was shown to be associated with AD neuropathology in the PHG brain regions.',
      data_used_to_support_target_selection:
        'Differential protein expression analysis, and Bayesian network analysis derived from proteomics profiling and WGS genotypes.',
      data_synapseid: 'syn17008057',
      study: 'MSBB',
      input_data: 'Genetics, Protein',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2020,
    },
  ],
  medianexpression: [
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 7.22549250226295,
      quartile1logcpm: 8.25778767262231,
      medianlogcpm: 8.65783306384424,
      meanlogcpm: 8.67822921701765,
      quartile3logcpm: 9.03802949087095,
      maximumlogcpm: 10.4142380183435,
      tissue: 'CBE',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 7.14223300774668,
      quartile1logcpm: 8.32840566635399,
      medianlogcpm: 8.80762326067988,
      meanlogcpm: 8.76242945984023,
      quartile3logcpm: 9.19358071314219,
      maximumlogcpm: 10.0652784802238,
      tissue: 'TCX',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 6.54927131897913,
      quartile1logcpm: 9.42875064236945,
      medianlogcpm: 9.76880015815198,
      meanlogcpm: 9.56963380060868,
      quartile3logcpm: 9.98652179920184,
      maximumlogcpm: 11.0867785895892,
      tissue: 'ACC',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 3.75330149083437,
      quartile1logcpm: 8.08399197480916,
      medianlogcpm: 9.22654506486567,
      meanlogcpm: 8.7516492110921,
      quartile3logcpm: 9.64834565315975,
      maximumlogcpm: 11.0867785895892,
      tissue: 'DLPFC',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 4.45648812246036,
      quartile1logcpm: 8.61673673314674,
      medianlogcpm: 9.63827007177671,
      meanlogcpm: 9.20767965842647,
      quartile3logcpm: 10.0352317821621,
      maximumlogcpm: 11.0307346887701,
      tissue: 'PCC',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 6.9225922182552,
      quartile1logcpm: 8.40526788696803,
      medianlogcpm: 9.16897860326366,
      meanlogcpm: 9.08904894721056,
      quartile3logcpm: 9.8602641380614,
      maximumlogcpm: 10.8176198993741,
      tissue: 'FP',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 8.1411622373572,
      quartile1logcpm: 9.32185036423896,
      medianlogcpm: 9.85111789837851,
      meanlogcpm: 9.78825590766753,
      quartile3logcpm: 10.1905335295928,
      maximumlogcpm: 11.4363890786455,
      tissue: 'IFG',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 5.80432490062941,
      quartile1logcpm: 8.34849621016103,
      medianlogcpm: 9.35468798616969,
      meanlogcpm: 9.19624463008193,
      quartile3logcpm: 10.0560861542771,
      maximumlogcpm: 11.0964398934874,
      tissue: 'PHG',
    },
    {
      ensembl_gene_id: 'ENSG00000178209',
      minimumlogcpm: 7.66844517113394,
      quartile1logcpm: 9.09921945493146,
      medianlogcpm: 9.61090389789751,
      meanlogcpm: 9.51496471073679,
      quartile3logcpm: 10.0599997750033,
      maximumlogcpm: 10.9003455990239,
      tissue: 'STG',
    },
  ],
  druggability: [
    {
      geneid: 'ENSG00000178209',
      sm_druggability_bucket: 3,
      safety_bucket: 4,
      abability_bucket: 3,
      pharos_class: 'Tbio',
      classification:
        'Targetable by structure: Structurally druggable protein, based on the presence of a druggable pocket in the protein (DrugEBIlity/CanSAR).',
      safety_bucket_definition:
        'More than two of: high off target gene expression, cancer driver, essential gene, associated deleterious genetic disorder, HPO phenotype associated gene, or black box warning on clinically used drug.',
      abability_bucket_definition:
        'Cell membrane-bound proteins. Highly accessible to antibody-based therapies, but potentially less so than secreted proteins or ECM components.',
    },
  ],
  nominations: 4,
};

export const mockGCTGene1: GCTGene = {
  ensembl_gene_id: 'ENSG00000147065',
  hgnc_symbol: 'MSN',
  tissues: [
    {
      name: 'ACC',
      logfc: -0.0144678061734664,
      adj_p_val: 0.893263674388766,
      ci_l: -0.133332670728704,
      ci_r: 0.104397058381771,
    },
    {
      name: 'CBE',
      logfc: -0.0751554371582435,
      adj_p_val: 0.530889774959758,
      ci_l: -0.247449816301241,
      ci_r: 0.0971389419847545,
    },
    {
      name: 'DLPFC',
      logfc: 0.0349940606540153,
      adj_p_val: 0.604655110745068,
      ci_l: -0.0564780903179806,
      ci_r: 0.126466211626011,
    },
    {
      name: 'FP',
      logfc: 0.235992067764791,
      adj_p_val: 0.0293685880983672,
      ci_l: 0.0813918568317721,
      ci_r: 0.390592278697809,
    },
    {
      name: 'IFG',
      logfc: 0.393613393602616,
      adj_p_val: 0.0000468370819411619,
      ci_l: 0.236604395922642,
      ci_r: 0.550622391282589,
    },
    {
      name: 'PCC',
      logfc: 0.0870992749746771,
      adj_p_val: 0.275931011190809,
      ci_l: -0.0292845799777793,
      ci_r: 0.203483129927133,
    },
    {
      name: 'PHG',
      logfc: 0.6626818751507,
      adj_p_val: 9.688467425132421e-14,
      ci_l: 0.506746460875332,
      ci_r: 0.818617289426068,
    },
    {
      name: 'STG',
      logfc: 0.421363564302165,
      adj_p_val: 0.0000165742209158385,
      ci_l: 0.26315969917125,
      ci_r: 0.57956742943308,
    },
    {
      name: 'TCX',
      logfc: 0.450452869099202,
      adj_p_val: 0.00000476005829762979,
      ci_l: 0.275315930793305,
      ci_r: 0.625589807405099,
    },
  ],
  nominations: {
    count: 4,
    year: 2018,
    teams: ['Chang Lab', 'Emory', 'MSSM', 'MSSM'],
    studies: ['ROSMAP', 'Kronos', 'MSBB', 'ACT', 'BLSA', 'Banner'],
    inputs: ['Genetics', 'RNA', 'Protein'],
    programs: ['Community Contributed', 'AMP-AD'],
    validations: [
      'validation studies ongoing',
      'not prioritized for experimental validation',
    ],
  },
  associations: [3, 4],
};

export const mockNominatedGene1: Gene = {
  _id: '628ea1be0e8d04279fdbaa26',
  ensembl_gene_id: 'ENSG00000147065',
  hgnc_symbol: 'MSN',
  nominatedtarget: [
    {
      source: 'Community',
      team: 'Chang Lab',
      rank: '',
      ensembl_gene_id: '',
      hgnc_symbol: '',
      target_choice_justification: '',
      predicted_therapeutic_direction: '',
      data_used_to_support_target_selection: '',
      data_synapseid: '',
      study: 'ROSMAP, Kronos, MSBB',
      input_data: 'Genetics, RNA, Protein',
      validation_study_details: '',
      initial_nomination: 2019,
    },
    {
      source: 'AMP-AD',
      team: 'Emory',
      rank: '',
      ensembl_gene_id: '',
      hgnc_symbol: '',
      target_choice_justification: '',
      predicted_therapeutic_direction: '',
      data_used_to_support_target_selection: '',
      data_synapseid: '',
      study: 'ACT, BLSA, Banner',
      input_data: 'Protein',
      validation_study_details: 'validation studies ongoing',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'MSSM',
      rank: '',
      ensembl_gene_id: '',
      hgnc_symbol: '',
      target_choice_justification: '',
      predicted_therapeutic_direction: '',
      data_used_to_support_target_selection: '',
      data_synapseid: '',
      study: 'MSBB',
      input_data: 'Genetics, RNA',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2018,
    },
    {
      source: 'AMP-AD',
      team: 'MSSM',
      rank: '',
      ensembl_gene_id: '',
      hgnc_symbol: '',
      target_choice_justification: '',
      predicted_therapeutic_direction: '',
      data_used_to_support_target_selection: '',
      data_synapseid: '',
      study: 'MSBB',
      input_data: 'Genetics, Protein',
      validation_study_details: 'not prioritized for experimental validation',
      initial_nomination: 2020,
    },
  ],
  druggability: [
    {
      geneid: '',
      sm_druggability_bucket: 3,
      safety_bucket: 4,
      abability_bucket: 3,
      pharos_class: 'Tbio',
      classification:
        'Targetable by structure: Structurally druggable protein, based on the presence of a druggable pocket in the protein (DrugEBIlity/CanSAR).',
      safety_bucket_definition:
        'More than two of: high off target gene expression, cancer driver, essential gene, associated deleterious genetic disorder, HPO phenotype associated gene, or black box warning on clinically used drug.',
      abability_bucket_definition:
        'Cell membrane-bound proteins. Highly accessible to antibody-based therapies, but potentially less so than secreted proteins or ECM components.',
    },
  ],
  nominations: 4,
} as Gene;
